Classes/Domain/Repository/AbstractDemandRepository.php

```diff
@@ -72,58 +77,67 @@ public function findDemanded(DemandInterface $demand): QueryResultInterface
      */
     public function countByDemand(DemandInterface $demand): int
     {
+        $queryBuilder = $this->createDemandQueryBuilder($demand);
```

Comment:
Do we need here queryBuilder if it not used anywhere in function?

Reply:
No, not really, removed the function now, couldn't find anywhere it was used.